#! /bin/sh /usr/share/dpatch/dpatch-run
## 10warnings.dpatch by LaMont Jones <lamont@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix compiler warnings

@DPATCH@
diff -urNad --exclude=CVS --exclude=.svn ./misc-utils/ddate.c /tmp/dpep-work.BBs5Mx/util-linux/misc-utils/ddate.c
--- ./misc-utils/ddate.c	2004-09-06 15:12:40.000000000 -0600
+++ /tmp/dpep-work.BBs5Mx/util-linux/misc-utils/ddate.c	2006-01-15 18:25:14.000000000 -0700
@@ -296,6 +296,7 @@
     int cal[12] = { 31,28,31,30,31,30,31,31,30,31,30,31 };
     int dayspast=0;
 
+    memset(&funkychickens,0,sizeof(funkychickens));
     /* basic range checks */
     if (imonth < 1 || imonth > 12) {
 	    funkychickens.season = -1;
diff -urNad --exclude=CVS --exclude=.svn ./misc-utils/script.c /tmp/dpep-work.BBs5Mx/util-linux/misc-utils/script.c
--- ./misc-utils/script.c	2004-12-15 08:32:52.000000000 -0700
+++ /tmp/dpep-work.BBs5Mx/util-linux/misc-utils/script.c	2006-01-15 18:25:57.000000000 -0700
@@ -325,9 +325,9 @@
 		shname = shell;
 
 	if (cflg)
-		execl(shell, shname, "-c", cflg, 0);
+		execl(shell, shname, "-c", cflg, NULL);
 	else
-		execl(shell, shname, "-i", 0);
+		execl(shell, shname, "-i", NULL);
 
 	perror(shell);
 	fail();
